pantheon-bundle/src/main/java/com/redhat/pantheon/validation/helper/XrefValidationHelper.java

```diff
@@ -8,14 +8,19 @@
 
 public class XrefValidationHelper {
 
-    private static Map<String, List<String>> xRefs = new HashMap<>();
+    private static List<String> xRefs;
```

Comment:
can't initialization be done here?

Reply:
@aprajshekhar in that case, if we have re upload scenarios, we have multiple duplicate elements corresponding to document, and validation nodes gets duplicated.